Styles/Style.css

```diff
@@ -0,0 +1,420 @@
+    font-size: 2rem;
+}
+
+.squiggle-main {
+    background-image: url(/Images/squiggle_line.svg);
+    height: 6px;
+    width: 3.5rem;
+}
+
+.outer-container img {
+    box-shadow: 0 10px 40px #7a7a7a;
+}
+
+.outer-container span{
+    display: flex;
+    flex: 2 1 auto;
+}
+
+.grey {
+    background-color: #F3F6F9;
+    z-index: -2;
+}
+
+.grey:after{
+    content: "work";
+    font-size: 25rem;
+    font-weight: 500;
+    text-align: center;
+    position: absolute;
+    color: rgb(255, 255, 255);
+    top: -15rem;
+    left: 1rem;
+    z-index: -1;
+    width: 100%;
+    height: 100%;
+    background: inherit;
+    bottom: 0;
+    transform-origin: right top;
+    transform: skewY(2deg);
+}
+
+.design-outer-container{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    gap: 2rem;
+    flex: 1 1 auto;
+    padding: 6rem 0;
+}
+
+.design-inner-container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    white-space: pre-line;
+    border: solid #5722CD 1px;
+    padding: 3rem 2rem;
+    flex: 1 1 auto;
+    background-color: white;
+    z-index: -1;
+}
+
+.design-inner-container h3{
+    color: #5722CD;
+    font-weight: 500;
+    margin-bottom: 1rem;
+}
+
+.design-inner-container p{
+    font-size: 12px;
+    color: #7a7a7a;
+}
+
+.service-container:after{
+    content: "services";
+    font-size: 15rem;
+    text-align: center;
+    font-weight: 500;
+    position: absolute;
+    color: rgb(242, 240, 255);
+    top: -6rem;
+    z-index: -2;
+}
+
+.rotate-element{
+    position: relative;
+    background-color: #F3F6F9;
+    height: 3rem;
+    padding: 3rem;
+    margin-bottom: 7rem;
+}
+
+.rotate-element:after {
+    content: " ";
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    color: #F3F6F9;
+    background: inherit;
+    top: 0;
+    z-index: -1;
+    transform-origin: left bottom;
+    transform: skewY(2deg);
+
+}
+/* --------------- 4.4 Notification ------------------- */
+
+.notification-purple{
+    background-image: linear-gradient(to right, #4845E4, #5722CD);
+    position: relative;
+}
+
+.notification-purple:after{
+    content: '';
+    width: 100%;
+    height: 100%;
+    position: absolute;
+    background: inherit;
+    z-index: -1;
+    bottom: 0;
+    transform-origin: right top;
+    transform: skewY(2deg);
+}
+
+.notification-inner-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 2rem;
+    flex: 1 1 auto;
+}
+
+.notification-inner-container h4{
+    font-family: 'Merriweather', serif;
+    font-size: 2rem;
+    color: white;
+    font-weight: 500;
+}
+
+.notification-inner-container p{
+    color: white;
+}
+
+.input-container{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    flex: 1 1 auto;
+}
+
+.input-container input{
+    display: flex;
+    flex: 1 1 auto;
+    width: 100%;
+    padding: 1rem 0 1rem 1rem;
+    border-radius: 0.3rem;
```

Comment:
Controleer altijd of je rem waardes overeenkomen met hele pixels. Dit komt namelijk neer op een kommagetal. Dat is niet lekker voor een browser om te renderen (laat staan, schalen)

Reply:
oh oké, kijk dat wist ik niet, weer wat geleerd :)